crates/marketplace/src/service.rs

```diff
@@ -375,6 +378,13 @@ where
             .try_collect()
             .await
     }
+
+    async fn txn_status(
+        &self,
+        _txn_hash: Commitment<<Types as NodeType>::Transaction>,
+    ) -> Result<TransactionStatus, BuildError> {
+        todo!("Implement this function");
```

Comment:
Can we return an Error here instead of panicking? I know we don't have this deployed yet, but I'd rather not panic in non-testing code anyways

Reply:
[fffda60](https://github.com/EspressoSystems/marketplace-builder-core/pull/236/commits/fffda60343be9b41b4145c0b2fb4227de1ac6841).